configuration/NI.Analysis.targets

```diff
@@ -0,0 +1,110 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- TODO: This location of 'content' needs to get updated once we build our package if we decide to put the files elsewhere, outside of a 'content' directory -->
+    <CodeAnalysisRuleSetDirectory>$(PkgNI_DotNet_Analyzers)\content</CodeAnalysisRuleSetDirectory>
+    <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+  </PropertyGroup>
+
+	<PropertyGroup>
+		<!-- this would more correctly be: assemblyname contains .Test. We have an issue with test utilities projects, and test.testutilities projects. -->
```

Comment:
We should document the behavior in these properties somewhere in the README. That way people who use these rules know when some rules apply and others don't.

Reply:
Yes, I agree that we need to add more documentation to the README. I'll make an action item to do that in a followup submission. I already have a work item for someone to help with the README in general

Reply:
I have a task in my backlog to add documentation on the different rulesets and which to use when and what behaviors are differenet.